server/mlabns/tests/test_tool_fetcher.py

```diff
@@ -40,45 +68,51 @@ def insertSliverTool(self, tool_id, site_id=None, status_ipv4=None,
         tool.latitude = latitude
         tool.longitude = longitude
         tool.country = country
-        tool.put()
+        self.created_tools.append(tool)
```

Comment:
IPv4 and IPv6 are both in the same SliverTool entry, but each slice at each site has its own SliverTool. You can see the full list here:

http://mlab-nstesting.appspot.com/admin/sliver_tools

The information is populated from Nagios with the code in server/mlabns/handlers/update.py